README.md

```diff
@@ -31,19 +31,19 @@ Our [build server](http://teamcity.monogame.net/?guest=1) builds, tests, and pac
 
 We support a growing list of platforms across the desktop, mobile, and console space.  If there is a platform we don't support, please [make a request](https://github.com/MonoGame/MonoGame/issues) or [come help us](CONTRIBUTING.md) add it.
 
-- Desktop PCs
- * Windows Store Apps (8, 8.1 and 10)
- * Windows (OpenGL & DirectX)
- * Linux (OpenGL)
- * Mac OS X (OpenGL)
-- Mobile Devices
- * Android (OpenGL)
- * iOS (OpenGL)
- * Windows Phone (8, 8.1 and 10)
-- Consoles (for registered developers)
- * PlayStation 4
- * PlayStation Vita
- * Xbox One (both UWP and XDK)
+ * Desktop PCs
+   * Windows Store Apps (8, 8.1 and 10)
+   * Windows (OpenGL & DirectX)
+   * Linux (OpenGL)
+   * Mac OS X (OpenGL)
+ * Mobile Devices
+   * Android (OpenGL)
+   * iOS (OpenGL)
```

Comment:
What about saying "iPhone/iPad/iTV" to make it clear we support all these devices?